src/index.html

```diff
@@ -26,10 +26,14 @@
     <!-- ! DO NOT REMOVE THIS COMMENT, WE NEED ITS CLOSING MARKER -->
   </head>
 
-  <body>
+  <body style="background: url('./src/assets/img/buddhanexus.jpg') cover;">
```

Comment:
please don't use inline styles. This can be styled with a class or tag selector, preferably in a `<style>` tag inside `<head>`.